app/static/src/app/components/WodinPlotDataSummary.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div v-if="hasData" hidden class="wodin-plot-data-summary">
+    <div class="wodin-plot-data-summary-series" v-for="(series, index) in data" :key="index"
+         :name="series.name"
+         :count="series.x?.length"
+         :x-min="Math.min(...series.x)"
+         :x-max="Math.max(...series.x)"
+         :y-min="Math.min(...series.y)"
+         :y-max="Math.max(...series.y)"
+         :mode="series.mode"
```

Comment:
this is always lines, no? I guess it's pretty harmless to keep though!

Reply:
It for sensitivity summary, but this just makes it consistent with the summary in the generic `WodinPlot`.